llvm/lib/Passes/PassRegistry.def

```diff
@@ -804,3 +804,31 @@ LOOP_PASS_WITH_PARAMS(
     },
     parseLoopUnswitchOptions, "nontrivial;no-nontrivial;trivial;no-trivial")
 #undef LOOP_PASS_WITH_PARAMS
+
+#ifdef MODULE_CALLBACK
+MODULE_CALLBACK("PipelineStartCallbacks", invokePipelineStartEPCallbacks)
```

Comment:
nit: this doesn't follow naming conventions in this file for passes, this should probably be something like `callbacks-pipeline-start`